flaml/ml.py

```diff
@@ -64,6 +65,7 @@ def sklearn_metric_loss_score(
         y_true: A 1d numpy array of the true labels
         labels: A 1d numpy array of the unique labels
         sample_weight: A 1d numpy array of the sample weight
+        multi_class: 'ovr' (one-vs-rest) or 'ovo' (one-vs-one)

```

Comment:
The default value is set to None. Could you document the expected behavior when it's None?